spec/factories/apply_applications.rb

```diff
@@ -23,8 +23,12 @@
       state { "importable" }
     end
 
-    trait :imported do
-      state { "imported" }
+    trait :importable do
+      state { "importable" }
+    end
+
+    trait :non_importable_duplicate do
+      state { "non_importable_duplicate" }
```

Comment:
Pretty sure it out of the box experience 
https://thoughtbot.github.io/factory_bot/traits/enum.html